tests/IntegrationTests/EULex.IntegrationTests.csproj

```diff
@@ -2,13 +2,13 @@
   <PropertyGroup>
     <RootNamespace>EULex.IntegrationTests</RootNamespace>
     <AssemblyName>EULex.IntegrationTests</AssemblyName>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net48</TargetFramework>
```

Comment:
With this I'm not able to run the tests on Linux (with dotnet 6.0).
I guess this should be the same target framework(s) as for the console sample app

Reply:
I was running on Windows when I ran the test, so forgot that `net48` won't run on other platforms. I agree, using the same target framework as the console app would be the best solution (`net6.0`).